apps/openchallenges/challenge-service/src/main/java/org/sagebionetworks/openchallenges/challenge/service/service/ChallengeAnalyticsService.java

```diff
@@ -14,9 +14,9 @@ public ChallengesPerYearDto getChallengesPerYear() {
     List<String> years =
         Arrays.asList(
             "2007", "2008", "2009", "2010", "2011", "2012", "2013", "2014", "2015", "2016", "2017",
-            "2018", "2019", "2020", "2021", "2022", "2023");
+            "2018", "2019", "2020", "2021", "2022", "2023", "2024");
     List<Integer> challengeCounts =
-        Arrays.asList(6, 9, 13, 17, 23, 29, 34, 41, 49, 59, 86, 97, 116, 135, 183, 242, 302);
+        Arrays.asList(6, 9, 13, 17, 23, 29, 34, 41, 49, 59, 86, 97, 116, 135, 184, 243, 305, 315);
```

Comment:
It looks like total 13 challenges were added (or dates were defined): 1 in 2021, 2 in 2023, 10 in 2024. Is it correct?

Reply:
In this PR, only 7 were added (3 do not have dates). The extraneous counts are probably from me forgetting to update the plot in the previous DB updates.